Source/WebCore/page/ElementTargetingController.cpp

```diff
@@ -784,8 +813,22 @@ static inline std::optional<IntRect> inflatedClientRectForAdjustmentRegionTracki
     return { inflatedClientRect };
 }
 
+static bool shouldIgnoreExistingVisibilityAdjustments(const TargetedElementRequest& request)
+{
+    return std::holds_alternative<String>(request.data);
+}
+
 Vector<TargetedElementInfo> ElementTargetingController::findTargets(TargetedElementRequest&& request)
 {
+    Vector<std::unique_ptr<ClearVisibilityAdjustmentForScope>> clearVisibilityAdjustmentScopes;
```

Comment:
This can just be a vector of values, right?

Reply:
Hmm, I think it's possible but likely a bit ugly since this RAII class is non-copyable. I'll try it out and see...

Thanks for the review!